RevenueCatUI/PaywallView.swift

```diff
@@ -209,6 +209,12 @@ public struct PaywallView: View {
                                 guard Purchases.isConfigured else {
                                     throw PaywallError.purchasesNotConfigured
                                 }
+                                
+                                if Purchases.shared.attribution.isThereUnsyncedAttributes {
+                                    _ = try? await Purchases.shared.syncAttributesAndOfferingsIfNeeded()
```

Comment:
you choose not to throw here? don't have a strong opinion

Reply:
Yes because I did not want this to be a blocker if it fails. Better have the wrong paywall than nothing